README.md

```diff
@@ -2,3 +2,39 @@
 Community contributions for STAT GR 5702 Fall 2019
 
 This is an amazing class.
+
+## Setup:
+
+1. Fork the cc19 repo (at https://github.com/jtr13/cc19). More information about how to achieve this on https://edav.info/github.html#branching-someone-elses-repo.
+
+2. Clone/download the repo to your local computers.
+
+3. Create and name a folder using your uni. If you are working as a team, the name is a concatenation of your and your partner's uni’s. 
+
+      For example, if your uni is xx1234 and your partner’s is yy4321, name the folder xx1234yy4321. This is the folder where ALL OF YOUR WORK will be in. Only one folder is needed for a team.
+
+
+
+## Important:
+
+If you are planning to have your bookdown page featured on the class bookdown. Please make the following modification on your rmd file before submission:
+
+1. Replace the YAML header with the following:
+
+      \<blank line\>
+      
+      \# \<Your project name\>
+
+Note that there is a whitespace between ‘#’ and a project name. Otherwise, the file will not be captured by the class index file.
+
+2. The second line of the text that part 1. asks you to add **must** be the only line with a single ‘#’. If there are parts in the document that need ‘#’s to bold the text, use more than 1 ‘#’s. 
+
+3. When the git repo is initiated, there will be a README file that lives outside the project folder you have created. **DO NOT ADD THIS README FILE when you commit.** It will create a conflict and we will not be able to merge your work into the class repo. 
+
```

Comment:
To be more clear here, say: Do not make any changes to the README outside of your folder. If you do accidentally, do not commit those changes.